download_data.sh

```diff
@@ -0,0 +1,18 @@
+BIO_SUBHEADINGS=("botany" "cell biology" "computational biology" "cryobiology" "developmental biology" "ecology" "exobiology" "genetics" "laboratory animal science" "microbiology" "natural history" "neurobiology" "parasitology" "photobiology" "radiobiology" "sociobiology" "synthetic biology" "zoology")
+INFORMATICS=("cheminformatics" "computational biology" "consumer health informatics" "dental informatics" "medical informatics" "nursing informatics" "public health informatics")
+ALGORITHMS=("artificial intelligence" "latent class analysis" "cellular automata")
```

Comment:
How did you choose these terms?

Reply:
I picked them while browsing the MeSH tree view. I'm currently working on more principled ways of picking headings